docs/reference/primitives/app/timer.md

```diff
@@ -25,9 +25,10 @@ A Timer is defined in [`convox.yml`](/configuration/convox-yml).
 
 | Name       | Required | Description                                                                                |
 | ---------- | -------- | ------------------------------------------------------------------------------------------ |
-| **command**  | **yes**  | The command to execute once the [Process](/reference/primitives/app/process) starts                               |
-| **schedule** | **yes**  | A cron formatted schedule for spawning the [Process](/reference/primitives/app/process). All times are UTC        |
-| **service**  | **yes**  | The name of the [Service](/reference/primitives/app/service) that will be used to spawn the [Process](/reference/primitives/app/process) |
+| **command**      | **yes**  | The command to execute once the [Process](/reference/primitives/app/process) starts                               |
+| **schedule**     | **yes**  | A cron formatted schedule for spawning the [Process](/reference/primitives/app/process). All times are UTC        |
+| **service**      | **yes**  | The name of the [Service](/reference/primitives/app/service) that will be used to spawn the [Process](/reference/primitives/app/process) |
+| **concurrency**  | **no**   | It specifies how to treat concurrent executions of a job that is created by this cron job. Check this [doc](https://kubernetes.io/docs/tasks/job/automated-tasks-with-cron-jobs/#concurrency-policy) for more info. |
```

Comment:
Can you also add that when not set the default is `forbid`?

Reply:
okay, I will explicit the default value. It was described in the docs but I agree with you.